ksql-serde/src/main/java/io/confluent/ksql/serde/FormatFactory.java

```diff
@@ -52,6 +54,7 @@ public static Format fromName(final String name) {
     switch (name) {
       case AvroFormat.NAME:       return AVRO;
       case JsonFormat.NAME:       return JSON;
+      case JsonSchemaFormat.NAME: return JSON_SR;
```

Comment:
nit: The naming `JsonSchemaFormat` doesn't make it immediately obvious how this is different to `JsonFormat`. Maybe `SchemaRegistryJsonFormat`?

Reply:
i'm following convention in the Converters (`JsonConverter` vs `JsonSchemaConverter`)